### PR TITLE
Refactor OpenZeppelin Relayer integration

### PR DESCRIPTION
**Pull request by Dzejkop**

### CODE REVIEW

File: src/contracts/mod.rs [hunk 1]
```diff
@@ -123,7 +124,7 @@ impl IdentityManager {
     }
 
     #[instrument(level = "debug", skip_all)]
-    pub async fn is_owner(&self) -> anyhow::Result<bool> {
+    pub async fn _is_owner(&self) -> anyhow::Result<bool> {
```
Comment (iamrecursion) [inline src/contracts/mod.rs:127]: Why?

File: src/contracts/mod.rs [hunk 2]
```diff
@@ -205,7 +206,7 @@ impl IdentityManager {
     }
 
     #[instrument(level = "debug", skip_all)]
-    pub async fn assert_valid_root(&self, root: Field) -> anyhow::Result<()> {
+    pub async fn _assert_valid_root(&self, root: Field) -> anyhow::Result<()> {
```
Comment (iamrecursion) [inline src/contracts/mod.rs:209]: Why?

File: src/contracts/mod.rs (hunk 1 above)
Comment (Dzejkop) [reply src/contracts/mod.rs:127]: They were unused (with and without `oz-provider` feature) which was throwing compilation warnings

Comment (gswirski) [reply src/contracts/mod.rs:127]: Let's ✂️ it then. :)

File: src/contracts/mod.rs (hunk 2 above)
Comment (gswirski) [reply src/contracts/mod.rs:209]: Let's ✂️ it then.

**Review by gswirski — APPROVED** [review]
👍

**Pull request merged**